lib/AST/Decl.cpp

```diff
@@ -4841,14 +4841,26 @@ findProtocolSelfReferences(const ProtocolDecl *proto, Type type,
     }
   }
 
+  const auto sselfTy = proto->getSelfInterfaceType();
```

Comment:
Q: is `sselfTy` a typo or does it carry some specific meaning?

Reply:
On second thought, it seems like another typo somewhere had fooled me into thinking this is a way to indicate that a leading letter should be read as a capital in lower camel case, so let us consider it a typo.